privaterelay/settings.py

```diff
@@ -660,62 +663,44 @@ def _get_initial_middleware() -> list[str]:
     "usd": {
         "en": {
             "monthly": {
-                "id": "price_1LXUenJNcmPzuWtRw3rhjQNP",
-                "price": "$?.??",
+                "id": PHONE_PLAN_ID_US_MONTHLY,
+                "price": "$4.99",
             },
             "yearly": {
-                "id": "price_1LXUhcJNcmPzuWtRHUFHVk12",
-                "price": "$?.??",
+                "id": PHONE_PLAN_ID_US_YEARLY,
+                "price": "$3.99",
             },
         },
-        # TODO: Get plan ID for Canada
-        # "ca": {
-        #     "monthly": {
-        #         "id": "",
-        #         "price": "$?.??",
-        #     },
-        #     "yearly": {
-        #         "id": "",
-        #         "price": "$?.??",
-        #     },
-        # },
     },
 }
 PHONE_PLAN_COUNTRY_LANG_MAPPING = {
     "us": {
         "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
     },
-    # Commented out; no plan ID known yet
-    # "ca": {
-    #     "en": PHONE_PLAN_ID_MATRIX["usd"]["ca"],
-    # },
+    "ca": {
+        "en": PHONE_PLAN_ID_MATRIX["usd"]["en"],
```

Comment:
Should this be `"cad"`?

Reply:
Unfortunately no, plans will be in USD for Canadians.

Reply:
What's even weirder is that people in the UK have to deal with this too! (But to be fair, at least they had the option of adopting the Euro :P)